### ISSUE TITLE
Confirmation dialogue for reboot/shutdown

### ISSUE THREAD

**Issue by wernerfred** [sddm/sddm#901]
In addtition to #342 it seems completly necessary to add a confirmation dialogue when pressing shutdown or reboot on the login screen.
Is there any chance to implement such function with Qt Quick or are changes in sddm necessary to make this working?

**Comment by davidedmundson** [comment]
Yes, it can be done purely within QtQuick, the KDE breeze theme does.

**Comment by wernerfred** [comment]
I just installed the breeze theme (apt install sddm-theme-breeze) and set it to the current theme.
After log out i pressed shutdown and the pc directly shutted down without warning or confirmation required. 
Any ideas what i'm doing wrong?
